db/index.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/photos', { useNewUrlParser: true });
+
```

Comment:
In the case of deployment, I suggest using the environment URI -
for example: process.env.MONGODB_URI